.github/workflows/mosaiq-db-tests.yml

```
@@ -63,9 +63,9 @@ jobs:
           pip install --upgrade pip setuptools
 
       - name: Install Poetry
-        uses: snok/install-poetry@v1.1.1
+        uses: snok/install-poetry@v1
```

```suggestion
        uses: snok/install-poetry@v1.3.1
```

It's worth pinning everything in the CI... (unless we are explicitly testing our library unpinned as in the `updates` test). We don't want things to change from passing to failing unless we explicitly change something, as much as is possible...